packages/block-editor/src/components/editor-styles/index.js

```diff
@@ -95,7 +113,7 @@ function EditorStyles( { styles, scope } ) {
 				.map( ( style ) => style.assets )
 				.join( '' ),
 		];
-	}, [ styles, overrides, scope ] );
+	}, [ styles, styleOverrides, scope ] );
```

Comment:
Won't we still need `overrides` still in the dependencies array?

Reply:
Oh yeah, missed that. Ta.

Updated.